libs-scala/resources-grpc/src/main/scala/com/daml/resources/grpc/ServerResourceOwner.scala

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.resources.grpc
+
+import com.daml.resources.{AbstractResourceOwner, HasExecutionContext, Resource}
+import io.grpc.{Server, ServerBuilder}
+
+import scala.concurrent.Future
+import scala.concurrent.duration.FiniteDuration
+
+private[grpc] final class ServerResourceOwner[Context: HasExecutionContext](
```

Comment:
Please make `com.daml.platform.apiserver.GrpcServer` use this.

Reply:
Addressed in https://github.com/digital-asset/daml/pull/8604/commits/14aa2b2960c88c72c3439c670af253b15b6696bf